### PR TITLE
Add 500 new words to Spanish-Catalan bidix (unknown Spanish module words)

### PR DESCRIPTION
**Pull request by albertonl**
See https://github.com/apertium/apertium-spa-cat/pull/2 for more information.

### CODE REVIEW

File: apertium-spa.spa.dix [hunk 1]
```diff
@@ -80700,6 +80700,390 @@
...
+<e lm="lastimar" a="albertonl">   <i>lastim</i><par n="abandon/ar__vblex"/></e>
+<e lm="liofilizar" a="albertonl">   <i>liofili</i><par n="abalan/zar__vblex"/></e>
+<e lm="lloviznar" a="albertonl">   <i>llovizn</i><par n="abandon/ar__vblex"/></e>
+<e lm="machetear" a="albertonl">   <i>machete</i><par n="abandon/ar__vblex"/></e>
+<e lm="olfatear" a="albertonl">   <i>olfate</i><par n="abandon/ar__vblex"/></e>
+<e lm="pestañear" a="albertonl">   <i>pestañe</i><par n="abandon/ar__vblex"/></e>
+<e lm="rechinar" a="albertonl">   <i>rechin</i><par n="abandon/ar__vblex"/></e>
+<e lm="retozar" a="albertonl">   <i>reto</i><par n="abalan/zar__vblex"/></e>
+<e lm="retranquear" a="albertonl">   <i>retranque</i><par n="abandon/ar__vblex"/></e>
+<e lm="ungir" a="albertonl">   <i>ung</i><par n="diri/gir__vblex"/></e>
```
Comment (jaumeortola) [inline apertium-spa.spa.dix:81084]: Spanish verbs are written in two lines. Look:
`<e lm="abandonar">       <i>abandon</i><par n="abandon/ar__vblex"/></e>`
`<e lm="abandonar">       <p><l>abandón</l>   <r>abandon</r></p><par n="abandón/ar__vblex"/></e>`

**Review by xavivars — CHANGES_REQUESTED** [review]
Most verbs need to be changed, see @jaumeortola's comments

**Comment by albertonl** [comment]
I'm on it

**Comment by albertonl** [comment]
Ok. Done

File: apertium-spa.spa.dix [hunk 2]
```diff
@@ -57965,7 +57965,7 @@
 <e lm="etiquetar">       <p><l>etiquét</l>   <r>etiquet</r></p><par n="abandón/ar__vblex"/></e>
 <e lm="europeizar">      <i>europeiz</i><par n="abandon/ar__vblex"/></e>
 <e lm="europeizar">      <p><l>europeíz</l>  <r>europeiz</r></p><par n="abandón/ar__vblex"/></e>
-<e lm="evacuar">         <i>evacu</i><par n="evacu/ar__vblex"/></e>
+<e lm="evacuar">         <i>evacu</i><par n="abandon/ar__vblex"/></e>
```
Comment (xavivars) [inline apertium-spa.spa.dix]: It's missing the form with á

Comment (albertonl) [reply apertium-spa.spa.dix]: Sorry, I just wanted to change the "evacu/ar__vblex" verbs in my section but I wrongly changed that. I will fix it in a moment

**Comment by albertonl** [comment]
There already was the entry with:
`<e lm="evacuar">  <p><l>evacú</l><r>evacu</r></p><par n="abandón/ar__vblex"/></e>`
, so I just added
`<e lm="evacuar">  <i>evacu</i><par n="abandon/ar__vblex"/></e>`

@jaumeortola, could it be like this?

**Comment by jaumeortola** [comment]
> @jaumeortola, could it be like this?

The problem is fixed here: https://github.com/apertium/apertium-spa/commit/805d5b50f690ac5ee02cfcd9423b4d994384770a

It's a bit tricky. You can now remove "evacuar" in your pull request.

**Pull request merged**